### ISSUE TITLE
package to Fedora

### ISSUE THREAD

**Issue by TomasTomecek** [TomasTomecek/ansible-bender#69]

**Comment by gordonmessmer** [comment]
I'm working on this:
https://bugzilla.redhat.com/show_bug.cgi?id=1714377

**Comment by TomasTomecek** [comment]
Oh my, this is awesome! Thank you so much.

**Comment by gordonmessmer** [comment]
I'm happy to help.  

Running "pytest-3" in mock produces mostly failures.  It looks like that is expected, since mock environments don't have network access and this appears to attempt to pull images.

Is there a subset of tests that will run in mock, or should we disable tests when building for Fedora?

**Comment by TomasTomecek** [comment]
Yes, integration and functional tests have some serious requirements on the environment they are running in.

I just checked unit tests and they should be fine to run after the build.

**Comment by gordonmessmer** [comment]
[pytest-unit.txt](https://github.com/ansible-community/ansible-bender/files/3233792/pytest-unit.txt)

It looks like even the unit tests require network connectivity.  I'll see if I can get this through review with a package that supports tests, but disables them by default.

**Comment by TomasTomecek** [comment]
Since Fedora has CI for packages, I thought that's the preferred way to run tests instead of the %check section.

Feel free to disable them. I test Bender on Fedora exclusively anyway.

**Comment by gordonmessmer** [comment]
Keep an eye on the review bug I linked above.  We'll see what the reviewers want from the packaging process.

**Comment by gordonmessmer** [comment]
The package has been approved, and I've been sponsored.  The new repository is here:

https://src.fedoraproject.org/rpms/ansible-bender

I'm waiting on a Fedora 30 branch now.  I'll update the spec for the new release before I submit a package to bodhi.

**Comment by gordonmessmer** [comment]
Working on the 0.7.0 package, I notice that the tarballs on pypi and on github don't match.  Which release should we be using?

**Comment by gordonmessmer** [comment]
https://bodhi.fedoraproject.org/updates/FEDORA-2019-5d253fb4e5

**Comment by TomasTomecek** [comment]
> Working on the 0.7.0 package, I notice that the tarballs on pypi and on github don't match. Which release should we be using?

PyPI ideally. Github tarballs are generated by github automatically, the ones on PyPI are done by our automation.


Thanks you very much for working on this, Gordon.

Closing this then (feel free to ask more).
